src/main/java/openperipheral/integration/forestry/AdapterBeeHousing.java

```diff
@@ -61,24 +61,26 @@ public IIndividual getQueen(IBeeHousing beeHousing) {
 
 	@Asynchronous
 	@ScriptCallable(returnTypes = ReturnType.TABLE, description = "Get the full breeding list thingy. Experimental!")
-	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing) {
+	public List<Map<String, Object>> getBeeBreedingData(IBeeHousing housing, @Optionals @Arg(name = "showSecrets", type = ArgType.BOOLEAN) Boolean showSecrets) {
```

Comment:
I was complaining from a balance point of view. Adding a parameter to show them anyway doesn't help at all.